CMakeLists.txt

```diff
@@ -661,6 +661,7 @@ set(cxx-sources
   tcp_listener.cpp
   thread.cpp
   trie.cpp
+  radix_tree.cpp
```

Comment:
Is trie still used elsewhere? If not, it should be removed.

Reply:
no, keep it - I'll add a build flag for now, I don't want to do a new release with untested code

Reply:
It seems the only files that use the trie now are the benchmarking scripts (`unittests/unittest_trie.cpp` and `unittests/unittest_trie_gb.cpp`). If this code isn't needed, I can move it out-of-tree elsewhere, a separate branch in my libzmq fork maybe. Does that sound like a good idea?

Reply:
Alright - we could try to port the benchmarks to C++98, so we'll just need to think about renaming and moving them elsewhere (this was suggested by @sigiesec in another comment). It seems the only significant C++11 features being used are the `<chrono>` and `<random>` headers; maybe writing a custom timer class and using another RNG should be enough to make it C++98-only.